src/applications/ezr/tests/e2e/ezr-next-of-kin.cypress.spec.js

```diff
@@ -0,0 +1,101 @@
+import manifest from '../../manifest.json';
+import mockUser from './fixtures/mocks/mock-user';
+import mockPrefill from './fixtures/mocks/mock-prefill.json';
+import maxTestData from './fixtures/data/maximal-test.json';
+import featureToggles from './fixtures/mocks/mock-features.json';
+import {
+  fillTextWebComponent,
+  goToNextPage,
+  selectDropdownWebComponent,
+  selectYesNoWebComponent,
+} from './helpers';
+import { MOCK_ENROLLMENT_RESPONSE } from '../../utils/constants';
+import { advanceToNextOfKin } from './helpers/next-of-kin';
+
+const { data: testData } = maxTestData;
+
+describe('EZR TERA flow', () => {
+  beforeEach(() => {
+    cy.login(mockUser);
+    cy.intercept('GET', '/v0/feature_toggles*', featureToggles).as(
+      'mockFeatures',
+    );
+    cy.intercept('GET', '/v0/health_care_applications/enrollment_status*', {
+      statusCode: 200,
+      body: MOCK_ENROLLMENT_RESPONSE,
+    }).as('mockEnrollmentStatus');
+    cy.intercept('/v0/in_progress_forms/10-10EZR', {
+      statusCode: 200,
+      body: mockPrefill,
+    }).as('mockSip');
+  });
+
+  it('should add next of kins', () => {
```

Comment:
It looks like the new code at line 33 introduces a new linting warnings. Can we please clean these up before we merge this?